app/models.py

```diff
@@ -34,6 +34,8 @@
 from app.encryption import check_hash, hashpw
 from app.history_meta import Versioned
 
+TemplateType = Literal["sms", "email", "letter"]
```

Comment:
We already have
https://github.com/cds-snc/notification-api/pull/1624/files#diff-90c680eba43456b516da8b4c2573d467ae17d1b0ed4373549f2a593ced3616d5R43 

Probably just need one of these?

Reply:
Looks like that just links to this PR